.github/workflows/ci-unittests.yml

```diff
@@ -115,3 +118,66 @@ jobs:
           fail_ci_if_error: true
           files: coverage.xml
           flags: unittests
+
+  docker:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_REGISTRY: gitlab.kitware.com:4567/smqtk/smqtk-iqr
+      CUDA_DEVEL_IMAGE_TAG: 9.2-cudnn7-devel-ubuntu18.04
+      CUDA_RUNTIME_IMAGE_TAG: 9.2-cudnn7-runtime-ubuntu18.04
+      FAISS_VERSION: 1.6.2
+      SMQTK_TAG: latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      # Cache local python environment artifacts for the current python version
+      # and poetry lockfile hash.
+      - uses: actions/cache@v2
+        id: env-cache
+        with:
+          # Confirmed that the `.local` directory doesn't exist until the
+          # `pip install --user` is invoked below, so we can be confident that
+          # only our python stuff is being captured in this cache (intentional).
+          path: |
+            ~/.cache/pypoetry/virtualenvs/
+            ~/.local
+          key: python-${{ matrix.python-version }}-${{ hashFiles('poetry.lock') }}
+
+      # Create .env file since it is not created initially
+      - name: Create env file
+        run: |
+          touch .env
+          echo SMQTK_REGISTRY=${{ env.DOCKER_REGISTRY }} >> .env
+          echo CUDA_DEVEL_IMAGE_TAG=${{ env.CUDA_DEVEL_IMAGE_TAG }} >> .env
+          echo CUDA_RUNTIME_IMAGE_TAG=${{ env.CUDA_RUNTIME_IMAGE_TAG }} >> .env
+          echo FAISS_VERSION=${{ env.FAISS_VERSION }} >> .env
+          echo SMQTK_TAG=${{ env.SMQTK_TAG }} >> .env
+          cat .env
```

Comment:
Does the `.env` file we have checked in not get checked out here? If it does get checked out, should we `cd docker/` and use the parameterization there? In the case of running on master/schedule, we can even leave the tag as "latest" as the semantic meaning would be upheld (as opposed to more ephemeral images that should be tagged differently on PRs).

Reply:
Yeah, just checked that it does. I edited the action to use that instead of recreating it.